src/test/test-binfmt-util.c

```diff
@@ -0,0 +1,17 @@
+#include "binfmt-util.h"
+#include "tests.h"
+
+TEST(disable_binfmt) {
+        ASSERT_OK(disable_binfmt());
+}
+
+TEST(binfmt_mounted) {
+        ASSERT_OK(binfmt_mounted());
+}
+
+static int intro(void) {
+        log_show_color(true);
+        return EXIT_SUCCESS;
+}
+
```

Comment:
This is not necessary unless you really want to unconditionally colorize logs.
Please drop and use `DEFINE_TEST_MAIN(LOG_DEBUG);`.